### ISSUE TITLE
MNIST MLP 例子需要的 operators

### ISSUE THREAD

**Issue by zsdonghao** [tensorlayer/tensorlayer-chinese#47]
以TF为参考:

- `tf.Variable`
- `tf.nn.bias_add`
- `tf.matmul`
- `tf.nn.relu`
- `tf.nn.dropout`
- `tf.reduce_mean`
- `tf.nn.sparse_softmax_cross_entropy_with_logits`
- `tf.optimizers.Adam`

TF求导例子：

```python
optimizer = tf.optimizers.Adam(learning_rate=0.0001)
with tf.GradientTape() as tape:
            ## compute outputs
            _logits = MLP(X_batch, foo=1)
            ## compute loss and update model
            _loss = tl.cost.cross_entropy(_logits, y_batch, name='train_loss')

grad = tape.gradient(_loss, MLP.weights)
optimizer.apply_gradients(zip(grad, MLP.weights))
```
